_posts/2023-05-20-introduction-juanita.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: single
+title: "🎶 My Journey Through Scientific Python: The Lyrics 🎶
+ "
+excerpt: "Hi there, I'm Juanita. I recently joined pyOpenSci as a community lead to help with community building and outreach and this blogpost tells the story of how I got here. "
+author: "Juanita Gomez"
+author: "Juanita Gomez"
+permalink: /blog/journey-through-scientific-python.html
+header:
+  overlay_color: "#DD751C"
+categories:
+  - blog-post
+  - highlight
+  - community
+toc: false
+comments: true
+---
+
+## Introduction: About Me
+
+Hello, everyone! I'm Juanita, and I'm excited to share my story with you. I'm currently a computer science PhD student at UC Santa Cruz and the community manager of the Scientific Python project (which you will hear more about in a bit 👀). I’m originally from Colombia, and I completed my undergraduate studies in pure mathematics at Universidad Javeriana while also taking some music courses. I'm a passionate singer, and my favorite color is orange 🧡. I recently joined pyOpenSci as a community lead to help with community building and outreach (super excited about it!), so today I want to share my journey through my involvement in open-source scientific Python projects, which is basically how I got here. 😊
+
+<figure>
+    <a href="/images/introduction-juanita/juanita-guitar.png">
+    <img src="/images/introduction-juanita/juanita-guitar.png" style="max-width:100%" alt="A woman sitting on top of a hill in the dessert holding a guitar.">
+    </a>
+    <figcaption>This is a picture of me in the middle of a dessert <a href="https://www.youtube.com/watch?v=lFxc5NZDjuA">  playing my favorite song </a>. One of the reasons I love this picture (besides the amazing landscape) it's because it shows how happy it makes me singing. 
+    </figcaption>
+</figure>
+
+
+## Verse 1: The Spark
+
+One of my earliest contributions to the open-source world was with the [Spyder IDE](https://www.spyder-ide.org) when I worked as a Software Engineer at Quansight. I primarily focused on writing documentation and engaging in community outreach. My initial task was to update the documentation for Spyder 4, the latest release at that time, and make it more user-friendly.
```

Comment:
Our standard short-form descriptor for Spyder is "the scientific Python development environment", since it pretty succinctly (and hopefully clearly) describes what it is/what it does, and is the origin of the name. So you could say:

```suggestion
One of my earliest contributions to the open-source world was with [Spyder](https://www.spyder-ide.org), the scientific Python development environment, when I worked as a Software Engineer at Quansight. I primarily focused on writing documentation and engaging in community outreach. My initial task was to update the documentation for Spyder 4, the latest release at that time, and make it more user-friendly.
```

If you want a screenshot, you could use [this one](https://github.com/spyder-ide/spyder/blob/master/img_src/screenshot.png), the (relatively) up to date hero on our web page and docs that Juanita herself took (her username is actually visible in the file paths).